tests/e2e/tensor_ops/CMakeLists.txt

```diff
@@ -93,3 +93,36 @@ iree_check_single_backend_test_suite(
 )
 
 ### BAZEL_TO_CMAKE_PRESERVES_ALL_CONTENT_BELOW_THIS_LINE ###
+
+if(IREE_HIP_TEST_TARGET_CHIP MATCHES "^gfx")
+
+unset(IREE_HIP_TEST_COMPILER_FLAGS)
+list(APPEND IREE_HIP_TEST_COMPILER_FLAGS
+  "--iree-rocm-target-chip=${IREE_HIP_TEST_TARGET_CHIP}"
+)
+
+iree_check_single_backend_test_suite(
+  NAME
+    check_hip_stream
+  SRCS
+    "pack.mlir"
```

Comment:
This test failed on the mi250 bot:
https://github.com/iree-org/iree/actions/runs/9392984478/job/25868510287?pr=17587#step:7:216
```
[ RUN      ] module.static_pack_pad_transpose_inner_and_outer_dims_large
iree/runtime/src/iree/modules/check/module.cc:372: Failure
Failed
Expected equality of these values. Contents does not match.
```

I can't tell from just the CI output which elements differ though... there are too many elements and some are elided. I think we have a flag to print more?

Reply:
I actually don't know how to debug now because it is passing on my workstation. I have a w7900, but not MI250. I'll ask the question in our sync tomorrow.